src/components/TalentsListPage/SearchPanel/SearchPanel.module.scss

```
@@ -0,0 +1,35 @@
+.search-panel {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap-reverse;
+  margin-top: 80px;
+}
+
+.search-input {
+  color: aliceblue;
+  font-family: "Plus Jakarta Sans";
+  width: 288px;
```

I recommend that you don't use pixel-perfect, it can make things more difficult later on.
```suggestion
  width: 290px;
```